manifests/kustomize/third-party/mysql/mysql-deployment.yaml

```diff
@@ -15,11 +15,16 @@ spec:
       labels:
         app: mysql
     spec:
+      serviceAccountName: mysql
       containers:
-      - env:
+      - args:
+        - --ignore-db-dir=lost+found
```

Comment:
Can you add a comment refer to why we need it?

Reply:
Done